Tella/Domain/Entity/Uwazi/UwaziServer.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  UwaziServer.swift
+//  Tella
+//
+//  Created by Gustavo on 13/11/2023.
+//  Copyright © 2023 HORIZONTAL. All rights reserved.
+//
+
+import Foundation
+
+class UwaziServer : Hashable {
```

Comment:
I suggest to create UwaziServer and TellaReportServer classes that inherit from a Server class containing the shared parameters.